### ISSUE TITLE
Bad results, parameters

### ISSUE THREAD

**Issue by sepbos** [dcaffo98/path-planning-cnn#7]
Hi I'm re-opening an issue regarding the parameter tuning.
aside from batch size are there any other parameters that should be tuned?
What parameters were you using when you got the results shown in the pictures in the article?

**Comment by dcaffo98** [comment]
Hi @sepbos . The parameters are shown in `training.py`, e.g. batch size, learning rate, learning rate scheduler, etc...

**Comment by sepbos** [comment]
I see thank you for the response. I was curious about the value of the parameters. To be specific the batch size, learning rate and loss function.
